docs/index.md

```diff
@@ -31,28 +31,30 @@ to you with the software itself.
 - **Manufacturer's Address**: 17 Grampian Place Tatton NSW 2650 Australia
 - **Software as a Medical Device Identification**: {{device_name}} {{device_version}}
 
-### Overview
+### Difference to the open source version
 
-Radiotherapy AI's contour recommendations product is deployed onsite as a DICOM
-server. It is designed to sit between your centre's CT scanner and your
+This device corresponds to the software downloadable directly from
+<https://radiotherapy.ai/> (Regulated Medical Device). It does **not**
+correspond to the software downloadable from
+<https://github.com/RadiotherapyAI/RadiotherapyAI> (Open Source Unregulated
+Software).
+
+The Open Source Unregulated Software is provided in the hope that it might be
+useful but WITHOUT ANY WARRANTY. It is **not** intended for clinical use, and
+is instead intended for research use only.
+
+The Regulated Medical Device described by this documentation corresponds only
+to the product that is able to be downloaded from <https://radiotherapy.ai/>.
+This is the product that is intended to be able to be used for clinical use
+according to the intended purpose described within this documentation.
+
+### Workflow overview
+
+Radiotherapy AI's contour recommendations product is deployed on-site as a
```

Comment:
it might help to say
`product (Regulated Medical Device) is deployed`

Reply:
Yup good call. Will do so. I'll pull out into a separate issue.